service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -105,29 +117,36 @@ public OrderTypeAndDocument buildOrderTypeAndDocument(OrderTypeAndDocument typeA
         switch (caseData.getOrderTypeAndDocument().getType()) {
             case BLANK_ORDER:
                 orderTemplateBuilder
-                    .put("orderType", BLANK_ORDER)
                     .put("orderTitle", defaultIfNull(caseData.getOrder().getTitle(), "Order"))
                     .put("childrenAct", "Children Act 1989")
                     .put("orderDetails", caseData.getOrder().getDetails());
                 break;
             case CARE_ORDER:
                 orderTemplateBuilder
-                    .put("orderType", CARE_ORDER)
                     .put("orderTitle", "Care order")
                     .put("childrenAct", "Section 31 Children Act 1989")
                     .put("orderDetails", careOrderDetails(getChildrenDetails(caseData).size(),
                         caseData.getCaseLocalAuthority()));
                 break;
+            case SUPERVISION_ORDER:
+                orderTemplateBuilder
+                    .put("orderTitle", "Supervision order")
+                    .put("childrenAct", "Section 31 and Paragraphs 1 and 2 Schedule 3 Children Act 1989")
+                    .put("orderDetails", supervisionOrderDetails(getChildrenDetails(caseData).size(),
+                        caseData.getCaseLocalAuthority(), caseData.getOrderMonths()));
+                break;
             default:
+                throw new IllegalStateException("Unexpected value: " + caseData.getOrderTypeAndDocument().getType());
```

Comment:
minor: i would say UnsupportedOperationException

Reply:
👍 Was just the default generated one